client/signup/config/flows.js

```diff
@@ -10,17 +10,26 @@ import config from 'config';
 import stepConfig from './steps';
 import userFactory from 'lib/user';
 import { generateFlows } from 'signup/config/flows-pure';
+import { getPlansSlugs } from 'lib/plans';
 
 const user = userFactory();
 
 function getCheckoutUrl( dependencies ) {
 	return `/checkout/${ dependencies.siteSlug }?signup=1`;
 }
 
+function getPlansUrl( dependencies ) {
+	return `/plans/${ dependencies.siteSlug }?signup=1`;
+}
+
 function dependenciesContainCartItem( dependencies ) {
 	return dependencies.cartItem || dependencies.domainItem || dependencies.themeItem;
 }
 
+function dependenciesContainPlan( dependencies ) {
+	return getPlansSlugs().includes( dependencies.cartItem?.product_slug );
```

Comment:
Updating this to lodash `includes` still runs into an error on IE11. 

@Automattic/team-calypso @sgomes any ideas? IE is reporting a syntax error on the polyfill eval:

<img width="1250" alt="Screen Shot 2020-02-17 at 10 48 11 AM" src="https://user-images.githubusercontent.com/1270189/74680007-4e4dd000-5174-11ea-85ba-14e3ce15cfc3.png">

Reply:
🤔 now the IE canary is fine https://circleci.com/gh/Automattic/wp-calypso/602070?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link but local is not happy. Is this expected?

Reply:
If other folks can't repro this, it's non-blocking, but do keep a closer eye on any potential breakages during deploy.

Reply:
There appears to be an arrow function in that snippet, which IE 11 wouldn't understand, and it would definitely complain about. This may be an error or not, depending on how you're running your tests.

We generate two builds in Calypso: evergreen and fallback. The former should only ever be used in modern browsers, and it contains fewer polyfills and more modern syntax, for an overall smaller size. The latter should work in legacy browsers like IE 11.

When running an `npm run start` in development, we only generate an evergreen build, as most development happens with modern browsers. So attempting to access that with a browser like IE 11 will produce errors.

In production and calypso.live, we generate both builds, and we look at the browser's UA string to determine which one to serve. If arrow functions make it into the fallback build (you can check the resource path for the JS files to see which one you're getting), that's an error. If IE 11 gets served an evergreen build in production or calypso.live, that's also an error.

If you want a local build that you can test in IE 11, you'll need to generate a production build that also does the differential build serving with evergreen/fallback. I haven't done this in a while (since I just use `calypso.live`), but I believe you can do that with `CALYPSO_ENV=production NODE_ENV=production npm run start`.

Hope this helps!

Reply:
@sgomes That's pretty awesome!!